File: 0455-assign-cookies/0455-assign-cookies.cpp
class Solution {
public:
    int findContentChildren(vector<int>& g, vector<int>& s) {
        sort(g.begin(), g.end());
        sort(s.begin(), s.end());
        int cnt = 0;
        for(int i=0, j=0; i<s.size(); i++) {
            if (s[i] >= g[j]) {
                cnt++;
                j++;
                if (j == g.size())
                    break;
            }
        }
        return cnt;
    }
};

File: 0455-assign-cookies/0455-assign-cookies.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        g.sort()
        s.sort()
        j, content = 0, 0
        for i in range(len(s)):
            if s[i] >= g[j]:
                content+=1
                j += 1
                if j == len(g):
                    break
        return content
